vcl_templates/www.vcl.erb

```diff
@@ -250,7 +250,28 @@ sub vcl_recv {
       set req.http.GOVUK-ABTest-EducationNavigation = "A";
     }
   }
-  
+
+  <% ['SearchMatchLength'].each do |test_name| %>
+  if (req.http.User-Agent ~ "^GOV\.UK Crawler Worker") {
```

Comment:
This is just the `vcl_recv` half of the CDN config, which adds a header to the request that's sent to the origin service. To send the cookie to the user, we'll have to update `vcl_deliver` as well. Are we intentionally skipping that step while we implement the rummager logic?